code/AssetLib/glTF2/glTF2Asset.h

```diff
@@ -376,6 +376,23 @@ struct CustomExtension {
     }
 };
 
+//! Represents metadata in an glTF2 object
+struct Extras {
+    std::vector<CustomExtension> mValues;
+
+    inline bool HasExtras() const {
+        return mValues.size() != 0;
+    }
+
+    Extras() = default;
+    ~Extras() = default;
+
+    Extras(const Extras &other) :
```

Comment:
I guess this copy constructor will be generated automatically by the compiler. So if you are not interested in having a special implementation feel free to remove it.